src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -152,41 +157,109 @@
                 // whitespace after is preferred
                 "TYPECAST", "LITERAL_SYNCHRONIZED").collect(Collectors.toSet()));
-        TOKENS_IN_CONFIG_TO_IGNORE.put("SeparatorWrap", Stream.of(
+        CHECKSTYLE_TOKENS_IN_CONFIG_TO_IGNORE.put("SeparatorWrap", Stream.of(
                 // needs context to decide what type of parentheses should be separated or not
                 // which this check does not provide
                 "LPAREN", "RPAREN").collect(Collectors.toSet()));
-        TOKENS_IN_CONFIG_TO_IGNORE.put("NeedBraces", Stream.of(
+        CHECKSTYLE_TOKENS_IN_CONFIG_TO_IGNORE.put("NeedBraces", Stream.of(
                 // we prefer no braces here as it looks unusual even though they help avoid sharing
                 // scope of variables
                 "LITERAL_DEFAULT", "LITERAL_CASE", "LAMBDA").collect(Collectors.toSet()));
-        TOKENS_IN_CONFIG_TO_IGNORE.put("FinalParameters", Stream.of(
+        CHECKSTYLE_TOKENS_IN_CONFIG_TO_IGNORE.put("FinalParameters", Stream.of(
                 // we prefer these to be effectively final as to not damage readability
                 "FOR_EACH_CLAUSE", "LITERAL_CATCH").collect(Collectors.toSet()));
-        TOKENS_IN_CONFIG_TO_IGNORE.put("WhitespaceAround", Stream.of(
+        CHECKSTYLE_TOKENS_IN_CONFIG_TO_IGNORE.put("WhitespaceAround", Stream.of(
                 // we prefer no spaces on one side or both for these tokens
                 "ARRAY_INIT",
                 // these are covered by GenericWhitespaceCheck
                 "WILDCARD_TYPE", "GENERIC_END", "GENERIC_START").collect(Collectors.toSet()));
+
+        // google
+        GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("AnnotationLocation", Stream.of(
+                // state of the configuration when test was made
+                "TYPECAST", "DOT", "TYPE_ARGUMENT", "ANNOTATION_DEF", "LITERAL_NEW",
+                "LITERAL_THROWS", "PARAMETER_DEF", "IMPLEMENTS_CLAUSE", "ANNOTATION_FIELD_DEF")
+                .collect(Collectors.toSet()));
+        GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("AbbreviationAsWordInName", Stream.of(
+                // state of the configuration when test was made
+                "ENUM_CONSTANT_DEF").collect(Collectors.toSet()));
+        GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("NoLineWrap", Stream.of(
+                // state of the configuration when test was made
+                "METHOD_DEF", "CTOR_DEF", "CLASS_DEF", "ENUM_DEF", "INTERFACE_DEF")
+                .collect(Collectors.toSet()));
+        GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("SeparatorWrap", Stream.of(
+                // state of the configuration when test was made
+                "RBRACK", "AT", "ELLIPSIS", "SEMI", "LPAREN", "RPAREN", "ARRAY_DECLARATOR")
+                .collect(Collectors.toSet()));
+        GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("NeedBraces", Stream.of(
+                // state of the configuration when test was made
```

Comment:
@romani Can you copy lambda in 4.5.1, I am not seeing it.
`NeedBraces` requires braces all the time, even if block is empty.
http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s4.1.1-braces-always-used Makes it seem like those are the only times blocks are required.